reflex/compiler/utils.py

```diff
@@ -257,18 +256,14 @@ def compile_custom_component(
     )
 
 
-def create_document_root(stylesheets: List[str]) -> Component:
+def create_document_root() -> Component:
     """Create the document root.
 
-    Args:
-        stylesheets: The list of stylesheets to include in the document root.
-
     Returns:
         The document root.
     """
-    sheets = [RawLink.create(rel="stylesheet", href=href) for href in stylesheets]
     return Html.create(
-        DocumentHead.create(*sheets),
+        DocumentHead.create(),
```

Comment:
Do we even need this now?

Reply:
Yes we still do, the page metadata arent loaded without it, which means the stylesheets wont render